tools/config.py

```diff
@@ -7,11 +7,11 @@
 
 # Update with the AMI id that you want to publish after TESTING
 
-PUBLISH_IMAGE_ID = 'ami-04d64f401811d1761'
+PUBLISH_IMAGE_ID = 'ami-090d4f51fad8aac21'
 
 # Update with the AMI name that you want to unpublish/delete worldwide
 
-DELETE_IMAGE_NAME = 'MeiliSearch-v0.21.0-Debian-10.3'
+DELETE_IMAGE_NAME = 'MeiliSearch-v0.23.0-Debian-10.3'
```

Comment:
```suggestion
DELETE_IMAGE_NAME = 'MeiliSearch-v0.23.1-Debian-10.3'
```

Reply:
This is the AMI we remove from AWS. We keep the last 2 versions of MeiliSearch AMI available and public so normally we should erase the v0.22.0 but as the v0.23.0 has the `Message: Too many open files (os error 24)` error I preferred to delete v0.23.0 instead of the v0.22.
![Screenshot 2021-11-17 at 17 21 52](https://user-images.githubusercontent.com/16155041/142239739-3bfff343-cd5d-47fc-9180-15a5c7decefe.png)